### ISSUE TITLE
GPSR i planowana blokada edycji

### ISSUE THREAD

**Issue by tomylee0** [allegro/allegro-api#10418]
Nie wiem czy dobrze zrozumiałem. Piszecie bowiem, że  w związku z nowymi przepisami nie będzie można edytować oferty jeśli nie będzie miała wypełnionego safetyInformation i responsibleProducer. A czy to nie jest tak, że przepisy dotyczą produktów zupełnie nowo wyprodukowanych po 13 grudnia i dopiero co wprowadzanych na rynek?

**Comment by creativepl-php** [comment]
Nie, GPSR obowiązuje niezależnie od tego czy produkty są nowe, używane, naprawione lub odnowione. Po prostu - każda nowa OFERTA od 13 grudnia ma być dostosowana do GPSR. Nieważne czy produkt jest sprzed 20 lat, oferta utworzona/edytowana po 13 grudnia = ma być GPSR.

**Comment by tomylee0** [comment]
Dzięki. Musiałem źle zrozumieć przepisy.

**Comment by jezewskiBNetrox** [comment]
> Musiałem źle zrozumieć przepisy.

@tomylee0 Nie zrozumiałeś przepisów źle. Allegro nie jest w stanie zweryfikować daty produkcji danego produktu, więc wymaga obowiązku z góry dla wszystkich ofert żeby załatać ten problem.

**Comment by creativepl-php** [comment]
@tomylee0 przepraszam, faktycznie użyłem tutaj dużego skrótu myślowego. Przepisy w zasadzie tego nie regulują, natomiast Allegro podejrzewam że idzie w ślad za Amazonem itp, stąd zapewne decyzja że po 13 grudnia wszystkie oferty mają być zgodne z gpsr. Błędem natomiast po stronie Allegro jest, że do istniejących już ofert będą wymagać gpsr przy edycji. Tego akurat regulacja nie wymaga, ale Allegro już tak. https://ecommercenews.pl/faq-dotyczace-obowiazkow-informacyjnych-i-wymagan-w-ramach-general-product-safety-regulation-gpsr-obowiazki-informacyjne-importerow-producentow-oraz-marek-wlasnych/

**Comment by Lukasz-Zurek** [comment]
@tomylee0 Aktywne oferty będziesz mógł edytować bez konieczności podawania dodatkowych informacji. Jeśli jednak produkt tego wymaga, jako sprzedający powinieneś dopilnować, aby oferty były zgodne z GPSR. Natomiast od 13 grudnia jako sprzedający musisz podać wymagane dane jeśli wystawiasz nową lub wznawiasz zakończoną ofertę.
